README.md

```diff
@@ -725,3 +725,60 @@ Error Connecting to auth service ...
 }
 ```
 
+
+
+
+### Get Importer Mappings
+
+This endpoint returns a list of available staging importer apps that have been whitelisted. This endpoint also
+returns a mapping between a list of files and predicted importer app.
+
+For example,
+ * if we pass in nothing we get a response with a list of apps, and no mappings
+ * if we pass in a list of files, such as ["file1.fasta", "file2.fq", "None"], we would get back a response
+ that maps to Fasta Importers and FastQ Importers, with a weight of 0 to 1 
+ which represents the probability that this is the correct importer for you.
+ * for files for which there is no predicted app, the return is a None/null value
+ * output type is not currently used for anything
+ * this endpoint is used to power the dropdowns for the staging service window in the Narrative
+
+**URL** : `ci.kbase.us/services/staging_service/importer_mappings`
+
+**local URL** : `localhost:3000/importer_mappings`
+
+**Method** : `POST`
+
+**Headers** : `Authorization: Not Required`
+
+## Success Response
+
+**Code** : `200 OK`
+
+**Content example**
+Example with passing in two files, one with a match, and one without
+```
+data = {"file_list": ["file1.txt", "file.zip"]}
+    async with AppClient(config, username) as cli:
+        resp = await cli.post(
+            "importer_mappings/", headers={"Authorization": ""}, data=data
```

Comment:
If the authorisation header is not present, what happens? If the Auth header isn't required at all, I'd omit it from the example.

Reply:
done